### ISSUE TITLE
Cannot use classes marked with @objc in IB in Xcode8 and Swift 3.

### ISSUE THREAD

**Issue by jbaez** [CosmicMind/Material#552]
After upgrading my project to Swift 3 and Material 2 I had an issue when compiling with messages similar to: "Unknown class _TtC8Material4View in Interface Builder file".
I turns out that in IB when setting the Custom Class it didn't show "Material" in the module section dropdown for some classes. 
The classes that had this issue also had the @objc attribute. I removed the attribute and then it showed "Material" in module section dropdown and compiled/worked without problems.

Classes that I found to have this issue are:
- Switch
- View
- Button

**Comment by danieldahan** [comment]
Can you send a sample to support@cosmicmind.io? The class names have changed and this may be the issue. We have samples in Storyboards that use classes with @objc. Thank you :)

**Comment by jbaez** [comment]
I created a simple sample project recreating the issue and sent it by email.

These is what I'm using:

- OSX El Capitan (10.11.6)
- Xcode 8 and Swift 3
- Cocoa pods 1.0.1

Thanks.

**Comment by danieldahan** [comment]
Excellent, thank you. I received your email. I will be taking a look shortly.

**Comment by danieldahan** [comment]
Looking now :)

**Comment by danieldahan** [comment]
You are correct. There is a warning. I believe it is because the @objc for those classes are not needed. I have removed them and there will be a release later today with the update. Thank you for bringing this to my attention :)

**Comment by markst** [comment]
I seem to be experiencing this issue with `TextView`.
Works fine if I remove `@objc(TextView)`. But I wonder if there's a better workaround to support objective-c & interface builder modules.

**Comment by danieldahan** [comment]
Hmmm... @markst can I see your setup code please? Thank you!

**Comment by markst** [comment]
I'll have to put together a demo project. But it would simply be to create a project - install Material. Create a storyboard with a `UITextView` then define `TextView` as the class type:
![image](https://cloud.githubusercontent.com/assets/274318/24945373/5f2bb588-1fa2-11e7-949c-b528fdddac27.png)

**Comment by danieldahan** [comment]
Cool, if you can do exactly that where the issue is reproducible, it would be great. Normally I would do it, but I have a lot to do at the moment. Thank you!

**Comment by markst** [comment]
Not sure if this is something further introduced in xcode 8.3

But it's also happening to `PageTabBarController`:
```
Unknown class _TtC8Material20PageTabBarController in Interface Builder file.
```
Again resolution is to remove the `@objc(PageTabBarController)`.

Afraid I can't provide any more details on reproducing, otherwise I'd fix the problem & PR myself.

**Comment by danieldahan** [comment]
I'll test and make a release for this today :) Thank you! @markst

**Comment by NikunjKumbhani** [comment]
Please Send me sample project which have integrate the material design source frame work 

i getting error Used of unresolved identifier 'objc_getAssociatedObject'

- Xcode 8 and Swift 3

Help me as soon as Possible.

**Comment by danieldahan** [comment]
@NikunjKumbhani There is an entire repo of samples https://github.com/CosmicMind/Samples :)

**Comment by NikunjKumbhani** [comment]
Thanks A lot @danieldahan

**Comment by danieldahan** [comment]
You can find a fix for this in [Material 2.7.0](https://github.com/CosmicMind/Material/releases/tag/2.7.0) :)
